src/main/views/webpack/css-template.njk

```diff
@@ -0,0 +1 @@
+<link href="/<%= htmlWebpackPlugin.files.css[0].substr(htmlWebpackPlugin.files.publicPath.length) %>" rel="stylesheet">
```

Comment:
Does this mean we don't need to update the version each time? :O :O great!

Reply:
that right made it the same as fact-admin